superset/db_engine_specs/bigquery.py

```diff
@@ -366,6 +376,21 @@ def df_to_sql(
 
         pandas_gbq.to_gbq(df, **to_gbq_kwargs)
 
+    @classmethod
+    def _get_client(cls, engine: Engine) -> Any:
+        """
+        Return the BigQuery client associated with an engine.
+        """
+        if not dependencies_installed:
```

Comment:
It looks like this is just catching a formerly uncaught exception, is that right?

Reply:
nm, I see where it was caught before with the ImportError